tests/conftest.py

```diff
@@ -128,7 +129,13 @@ def app(env_vars):
 @pytest.fixture(scope='session')
 def db_tables_session_scope(db_engines):
     from cnxdb.contrib.pytest import db_tables
-    tables = db_tables(db_engines)
+    with warnings.catch_warnings():
```

Comment:
You pretty much reverted the changes from the previous commit.  Do we also want to revert setup.cfg to unignore `sqlalchemy.exc.SAWarning`?

Reply:
No, without the ignore the test runner fails hard (face plant). This is more or less manipulating the issue without resolving it, because it's not within scope for this changeset.